packages/xo-web/src/common/store/actions.js

```diff
@@ -66,3 +66,11 @@ export const markHubResourceAsInstalled = createAction(
   'MARK_HUB_RESOURCE_AS_INSTALLED',
   id => id
 )
+export const markRecipeAsCreating = createAction(
+  'MARK_RECIPE_AS_CREATING',
+  id => id
+)
+export const markRecipeAsCreated = createAction(
```

Comment:
Is "created" a correct wording? Is a recipe "being created" or does a recipe create something?

Reply:
A recipe is executed, it creates one or more VMs.

Reply:
`markRecipeAsCreating` ?
In the other hand :
`markRecipeAsFinishedCreating` ?